packages/marko-web-inquiry/translations.js

```diff
@@ -0,0 +1,10 @@
+const translations = {
+  en: {
+    firstNameLabel: 'First Name', surnameLabel: 'Last Name', emailLabel: 'Email Address', phoneLabel: 'Phone Number', companyLabel: 'Company Name', jobTitleLabel: 'Job Title', countryLabel: 'Country', countryPlaceholder: 'Select country...', zipLabel: 'ZIP/Postal Code', commentsLabel: 'Comment', submitLabel: 'Submit',
+  },
+  es: {
+    firstNameLabel: 'Primer nombre', surnameLabel: 'Apellidos', emailLabel: 'Correo electrónico', phoneLabel: 'Número telefónico', companyLabel: 'Nombre de compañía', jobTitleLabel: 'Título de trabajo', countryLabel: 'País', countryPlaceholder: 'Selecciona un país...', zipLabel: 'Código postal', commentsLabel: 'Comentarios', submitLabel: 'Enviar',
+  },
+};
+
+export default translations;
```

Comment:
When exporting a single item, don't define it as a variable (unless you need to re-use it elsewhere in the file). This should read
```js
export default {
  en: {
  // ...
```